flask/app/routes.py

```diff
@@ -311,24 +311,35 @@ def bulk_update():
             "notes",
         ],
     }
+    if request.content_type not in ["text/csv", "application/json"]:
+        return "Only Content Type 'text/csv' or 'application/json' Supported", 415
```

Comment:
This could be done as the final `else:` clause for the if/elif following

Reply:
good point, I actually thought the same since it seemed out of place at the beginning